pandas/io/formats/info.py

```diff
@@ -174,26 +302,26 @@ def render(
         show_counts: Optional[bool],
     ) -> None:
         """
-        Print a concise summary of a %(klass)s.
+        Print a concise summary of a {klass}.
```

Comment:
Thank you for the comment!
I tried that, but the problem is that in future I would like to share the docstring with ``Series.info()``.
If we put ``@doc`` with params pertaining to dataframes onto ``BaseInfo.render``, then how would we handle the same for series?

Reply:
The `@doc` decorator does some magic. Basically it renders the docstring (template) with the provided variables, and it stores it as the `__doc__`, so that's the docstring of the function/class. But also, it saves the original docstring before rendering the variables in something like `__doc_original__` (forgot the actual name), so if you reuse that docstring somewhere else with different variables, it'll work as "expected".